### ISSUE TITLE
Question for Norm of Loss

### ISSUE THREAD

**Issue by lumos1iu** [Jiahao-UTS/SLPT_Training#3]
I noticed that in the paper, the distance between outer eye corners that acts as a normalization factor.

The landmarks are already normalized by image size before fed in network. I would like to know what the purpose of another normalization is and if removing it has little effect on the results.

**Comment by lumos1iu** [comment]
![Uploading 2cf84291e9728bd702aa683aa01899f5.jpg…]()

**Comment by Jiahao-UTS** [comment]
Thanks for your interest. I just use this normalization to help me better observe the change of NME on the training set. I did not try a version without this normalization. Maybe you can try one ^_^. If it could improve the performance, please let me know. Thank you!
